spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java

```diff
@@ -485,4 +484,501 @@ public String getExperimental(String name) {
+				enabled == that.enabled &&
+				Objects.equals(transport, that.transport) &&
+				registryFetchIntervalSeconds == that.registryFetchIntervalSeconds &&
+				instanceInfoReplicationIntervalSeconds == that.instanceInfoReplicationIntervalSeconds &&
+				initialInstanceInfoReplicationIntervalSeconds == that.initialInstanceInfoReplicationIntervalSeconds &&
+				eurekaServiceUrlPollIntervalSeconds == that.eurekaServiceUrlPollIntervalSeconds &&
+				eurekaServerReadTimeoutSeconds == that.eurekaServerReadTimeoutSeconds &&
+				eurekaServerConnectTimeoutSeconds == that.eurekaServerConnectTimeoutSeconds &&
+				eurekaServerTotalConnections == that.eurekaServerTotalConnections &&
+				eurekaServerTotalConnectionsPerHost == that.eurekaServerTotalConnectionsPerHost &&
+				eurekaConnectionIdleTimeoutSeconds == that.eurekaConnectionIdleTimeoutSeconds &&
+				heartbeatExecutorThreadPoolSize == that.heartbeatExecutorThreadPoolSize &&
+				heartbeatExecutorExponentialBackOffBound == that.heartbeatExecutorExponentialBackOffBound &&
+				cacheRefreshExecutorThreadPoolSize == that.cacheRefreshExecutorThreadPoolSize &&
+				cacheRefreshExecutorExponentialBackOffBound == that.cacheRefreshExecutorExponentialBackOffBound &&
+				gZipContent == that.gZipContent &&
+				useDnsForFetchingServiceUrls == that.useDnsForFetchingServiceUrls &&
+				registerWithEureka == that.registerWithEureka &&
+				preferSameZoneEureka == that.preferSameZoneEureka &&
+				logDeltaDiff == that.logDeltaDiff &&
+				disableDelta == that.disableDelta &&
+				filterOnlyUpInstances == that.filterOnlyUpInstances &&
+				fetchRegistry == that.fetchRegistry &&
+				allowRedirects == that.allowRedirects &&
+				onDemandUpdateStatusChange == that.onDemandUpdateStatusChange &&
+				Objects.equals(proxyPort, that.proxyPort) &&
+				Objects.equals(proxyHost, that.proxyHost) &&
+				Objects.equals(proxyUserName, that.proxyUserName) &&
+				Objects.equals(proxyPassword, that.proxyPassword) &&
+				Objects.equals(backupRegistryImpl, that.backupRegistryImpl) &&
+				Objects.equals(eurekaServerURLContext, that.eurekaServerURLContext) &&
+				Objects.equals(eurekaServerPort, that.eurekaServerPort) &&
+				Objects.equals(eurekaServerDNSName, that.eurekaServerDNSName) &&
+				Objects.equals(region, that.region) &&
+				Objects.equals(registryRefreshSingleVipAddress, that.registryRefreshSingleVipAddress) &&
+				Objects.equals(serviceUrl, that.serviceUrl) &&
+				Objects.equals(fetchRemoteRegionsRegistry, that.fetchRemoteRegionsRegistry) &&
+				Objects.equals(availabilityZones, that.availabilityZones) &&
+				Objects.equals(dollarReplacement, that.dollarReplacement) &&
+				Objects.equals(escapeCharReplacement, that.escapeCharReplacement) &&
+				Objects.equals(encoderName, that.encoderName) &&
+				Objects.equals(decoderName, that.decoderName) &&
+				Objects.equals(clientDataAccept, that.clientDataAccept);
+	}
+
+	@Override
+	public int hashCode() {
+		return Objects.hash(this);
```

Comment:
@spencergibb: Thanks for pointing this out, I have missed that.  I will make sure that my upcoming changes (which will conclude the removal of Lombok) implement this properly.